### PR TITLE
Add Manufacturer filter to product list block

### PR DESCRIPTION
**Pull request by edbeeny**

### CODE REVIEW

File: src/CommunityStore/Product/ProductList.php [hunk 1]
```diff
@@ -306,6 +312,9 @@ public function finalizeQuery(\Doctrine\DBAL\Query\QueryBuilder $query)
         if ($this->featuredOnly) {
             $query->andWhere("pFeatured = 1");
         }
+        if ($this->FilterManufacturers) {
+            $query->andWhere("pManufacturer =  $this->FilterManufacturers " );
```
Comment (Mesuva) [inline src/CommunityStore/Product/ProductList.php]: This is best handled with a parameterized query, so:
`$query->andWhere("pManufacturer = ?")->setParameter($paramcount++, $this->manufacturer);`

File: blocks/community_product_list/form.php [hunk 2]
```diff
@@ -96,6 +97,21 @@ class="existing-select2 select2-select" autocomplete="off" style="width: 100%" p
...
+            <?php
+            $productmanufacturers = array("0" => t("None"));
+            foreach ($manufacturersList as $productmanufacturer) {
+                $productmanufacturers[$productmanufacturer->getID()] = $productmanufacturer->getName();
+            }
+            ?>
+            <?php if (!empty($productmanufacturer)) {
+            ?>
+            <div class="form-group">
+                <?= $form->label('mID', t('Filter by Product Manufacturer')); ?>
```
Comment (Mesuva) [inline blocks/community_product_list/form.php]: change to 'Filter by Brand / Manufacturer'  (to match what it's called on the edit product form)

**Comment by edbeeny** [comment]
Changes as requested, I will try and have a look at the  Product List Filter block later today/tomorrow.

**Comment by Mesuva** [comment]
Excellent - I think the Product List filter will be mostly cut and paste, but let me know if you need any help

**Comment by edbeeny** [comment]
I think that is what is required for the product filter block. Let me know if there are any changes to be made.

**Comment by Mesuva** [comment]
Looks good at a glance, nice one.
I'm away from the office again for a bit, but when i'll return (or when I have a quiet moment) I'll merge this in.

**Comment by Mesuva** [comment]
Apologies again for not getting to this - we've had a busy few weeks and I'm trying to finish up something to merge in before this. If you need it urgently merged please let me know

**Comment by edbeeny** [comment]
No rush for this to be merged. Thanks

**Pull request merged**